src/olympia/blocklist/cron.py

```diff
@@ -85,4 +87,10 @@ def auto_import_blocklist():
     if not waffle.switch_is_active('blocklist_auto_import'):
         log.info('Automatic import_blocklist cron job disabled.')
         return
-    call_command('import_blocklist')
+    with statsd.timer('blocklist.cron.import_blocklist'):
+        try:
+            call_command('import_blocklist')
+        except CommandError as err:
+            statsd.incr('blocklist.cron.import_blocklist.error')
```

Comment:
```suggestion
            statsd.incr('blocklist.cron.import_blocklist.failure')
```

We use this convention elsewhere.

Reply:
https://github.com/mozilla/addons-server/pull/14330/commits/0a2ee2a7c66e5a06d327763b53c45f974dc9fbca